### PR TITLE
[token] update transfer doc

### PR DESCRIPTION
**Pull request by areshand**
### Description
1. token transfer as one part of token lifecycle
2. update the token transfer documentation to clearly list 3 different mode with links
3. remove detail struct field documentation and should directly link to code comment

### CODE REVIEW

File: developer-docs-site/docs/concepts/coin-and-token/aptos-token.md [hunk 1]
```diff
@@ -181,60 +181,7 @@ The following tables describe fields at the struct level. For the definitive lis
...
-| Field | Type | Description |
-| --- | --- | --- |
-| `description` | String | A description for the token collection Eg: “Aptos Toad Overload” |
-| `name` | String | The collection name, which should be unique among all collections by the creator; the name should also be smaller than 128 characters, eg: “Animal Collection” |
-| `uri` | String | The URI for the collection; its length should be smaller than 512 characters |
-| `supply` | u64 | The number of different TokenData entries in this collection |
-| `maximum` | u64 | If maximal is a non-zero value, the number of created TokenData entries should be smaller or equal to this maximum If maximal is 0, Aptos doen’t track the supply of this collection, and there is no limit |
-| `mutability_config` | CollectionMutabilityConfig | Specifies which fields are mutable  |
-
+For struct field definion, fee
```
Comment (clay-aptos) [inline developer-docs-site/docs/concepts/coin-and-token/aptos-token.md:184]: This appears misspelled ("see?") and incomplete.

**Review by clay-aptos — APPROVED** [review]
Thank you, Bo! This LGTM with some small edits I made directly here in a distinct commit and one incomplete sentence I highlighted in comment.

File: developer-docs-site/docs/concepts/coin-and-token/aptos-token.md [hunk 2]
```diff
@@ -270,24 +217,23 @@ We provide `burn` and `burn_by_creator` functions for token owners and token cre
...
-## Token transfer
+### Token transfer
 
-To protect a user from receiving undesired NFTs, an Aptos user must be first offered an NFT, followed by the user claiming the offered NFTs. Then only these NFTs will be deposited in the user's token store. This is the default token transfer behavior. For example:
+We provide three different modes for transferring tokens between the sender and receiver.
 
-1. If Alice wants to send Bob an NFT, she must first offer Bob this NFT. This NFT is still stored under Alice’s account. 
-1. Only when Bob claims the NFT, this NFT will be removed from Alice’s account and stored in Bob’s token store. 
+Two-step transfer: to protect users from receiving undesired NFTs, they must be first offered NFTs, and then accept the offered NFTs. Then only the offered NFTs will be deposited in the users' token stores. This is the default token transfer behavior. For example:
+1. If Alice wants to send Bob an NFT, she must first offer Bob this NFT. This NFT is still stored under Alice’s account.
```
Comment (movekevin) [inline developer-docs-site/docs/concepts/coin-and-token/aptos-token.md:225]: Numbering looks wrong here

**Review by movekevin — APPROVED** [review]

File: developer-docs-site/docs/concepts/coin-and-token/aptos-token.md (hunk 2 above)
Comment (clay-aptos) [reply developer-docs-site/docs/concepts/coin-and-token/aptos-token.md:226]: This is automated:
https://github.com/aptos-labs/aptos-core/blob/46aec6b97c16a632a17461468cf3b40b4274d139/developer-docs-site/docs/concepts/coin-and-token/aptos-token.md

**Comment by clay-aptos** [comment]
Belatedly, is it worth exploring whether any of the removed information from the tables needs to go in the linked reference docs?

**Pull request merged**